.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules/
 bower_components/
 config/config.js
 rise-financial-version.js
+rise-financial.html
```

Comment:
@settinghead Nice, didn't know that was possible. @donnapep Agreed, so long as we can publish to contain those files I'm good with ignoring in git.